plugins/BEdita/API/config/routes.php

```diff
@@ -64,6 +64,13 @@
             ['_name' => 'status']
         );
 
+        // Sysinfo.
+        $routes->connect(
```

Comment:
Why not `/admin/sysinfo` ? The `/admin` prefix makes it more clear that this is admin stuff

`SysInfoController` should then be moved to `Admin` namespace

Reply:
ok for `/admin/sysinfo`